core/src/test/java/org/elasticsearch/ExceptionSerializationTests.java

```diff
@@ -745,7 +745,7 @@ public void testIds() {
         ids.put(120, org.elasticsearch.repositories.RepositoryVerificationException.class);
         ids.put(121, org.elasticsearch.search.aggregations.InvalidAggregationPathException.class);
         ids.put(122, null);
-        ids.put(123, org.elasticsearch.indices.IndexAlreadyExistsException.class);
```

Comment:
this doesn't work sorry... you can't add a new exception here it's a wire protocol break. Just reuse the 123 and add a comment that we renamed it